### PR TITLE
Issue #20 output any crash logs on macOS

### PR DESCRIPTION
**Pull request by ajmas**
This is not a fix, but just seeing if we can output crash logs on macOS in the meantime.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,6 +43,12 @@ after_success:
       bash <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/master/codecov) -a '-r';
     fi
 
+after_failure:
+  - if [ $TRAVIS_OS_NAME == osx ]; then
+      cd  ~/Library/Logs/DiagnosticReports
```
Comment (GilesBathgate) [inline .travis.yml]: You will need to add semicolons:
```
after_failure:
  - if [ $TRAVIS_OS_NAME == osx ]; then
      cd  ~/Library/Logs/DiagnosticReports;
      cat rapcad_*.crash;
    fi
```

**Comment by GilesBathgate** [comment]
Awaiting answer from SO:
https://stackoverflow.com/questions/59887716/travis-ci-how-can-the-macos-crash-log-be-made-available-as-an-artefact

**Comment by ajmas** [comment]
Yup, I posted that one. I might try to create a MacOS 13 VM on my Mac and compare environments. 

In the meantime, it could be useful to see how to do something equivalent on Linux.

**Comment by GilesBathgate** [comment]
The issue this PR attempted to investigate is now closed.

**Pull request closed without merging**